### ISSUE TITLE
After closing modal, it appears again.

### ISSUE THREAD

**Issue by louiswalch** [dimsemenov/PhotoSwipe#1127]
Encountering a strange problem. When I click the X to close modal, it closes, but then opens back up again and  shows the last image in the pswp element (but no background overlay). I have `history: false` set. Hard to explain, so here is a demonstration video:
https://youtu.be/xyUa_dAwxE8

**Comment by dimsemenov** [comment]
Please provide a [reduced test case](http://css-tricks.com/reduced-test-cases/) (isolated example) that reproduces the problem. You may create it based on CodePens in PhotoSwipe documentation: [raw gallery](http://codepen.io/dimsemenov/pen/gbadPv), [gallery with thumbnails](http://codepen.io/dimsemenov/pen/ZYbPJM), [custom HTML content in slides](http://codepen.io/dimsemenov/pen/MYexrm). If you're not comfortable with CodePen, you may export code from it via `Export -> Export .zip`, and upload test case to your server, or use any other online tool like [JSFiddle](https://jsfiddle.net) or [JSBin](https://jsbin.com/).

**Comment by shmkv90** [comment]
I followed these steps to reproduced a similar problem: 
1. Refresh the page with updating the cache.
2. Call up the image of the gallery until the entire page was loaded.

After closing the gallery I handle an exception in spotoswipe.js on 2938 line
`// remove reference to DOM elements, for GC`

So I wrapped this block in a try/catch section and handle it in this way
```
    $('.pswp__item').hide();
    $('.pswp__item').css('display', 'block');
```
